src/test/java/com/rackspace/ceres/app/services/MetadataServiceCachingTest.java

```diff
@@ -167,20 +181,45 @@ void confirmMaxCacheConfig() {
     final String tagV = randomAlphanumeric(5);
     final Map<String, String> tags = Map.of(tagK, tagV);
 
+    assertThat(seriesSetExistenceCache.synchronous().estimatedSize()).isEqualTo(0);
+
     Mono.from(
         metadataService.storeMetadata(tenant, seriesSetHash1, metricName, tags)
     ).block();
-    verify(opsForValue)
-        .setIfAbsent(String.format("seriesSetHashes|%s|%s", tenant, seriesSetHash1), "");
+
+    await().atMost(Duration.ofSeconds(1)).untilAsserted(() ->
+        assertThat(seriesSetExistenceCache.synchronous().estimatedSize()).isEqualTo(1)
+    );
 
     // store the same again, but it should hit cache and not redis
+    Mono.from(
+        metadataService.storeMetadata(tenant, seriesSetHash1, metricName, tags)
+    ).block();
+
+    await().atMost(Duration.ofSeconds(1)).untilAsserted(() ->
+        assertThat(seriesSetExistenceCache.synchronous().estimatedSize()).isEqualTo(1)
+    );
+
+    // store a different one to displace the one cache entry
     Mono.from(
         metadataService.storeMetadata(tenant, seriesSetHash2, metricName, tags)
     ).block();
+
+    await().atMost(Duration.ofSeconds(1)).untilAsserted(() ->
```

Comment:
i'm confused.  at this point, shouldn't hash1 and hash2 both be in the cache?

Reply:
i think that was my last question.   i've reviewed everything else.

Reply:
I should probably add an inline comment there. The cache size was configured to 1 up at the top

https://github.com/racker/ceres/pull/9/files#diff-dd8c8c8380d0d4d7c679ecb85ac0e4786cf3506a9867086d325d0e07133db330R60